src/components/CommentList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Comment } from './Comment';
+import { TypeComments } from '../types';
+
+export const CommentList = ({ comments }) => (
+  <>
+    <h2>Comments</h2>
+    {comments.map(comment => (
+      <Comment {...comment} key={comment.id} />
+    ))}
+  </>
+);
+
+CommentList.propTypes = {
+  comments: TypeComments.isRequired,
```

Comment:
```suggestion
  comments: PropTypes.arrayOf(TypeComments).isRequired,
```

Reply:
TypeComments is already array of objects, https://prnt.sc/wquglx should i change it?

Reply:
@Ruslanwp ok, but please next time store only a shape in the variable, and then add it to the component with arrayOf